custom_components/openevse/__init__.py

```
@@ -428,190 +429,153 @@ async def update_sensors(self) -> dict:
     async def websocket_update(self):
         """Trigger processing updated websocket data."""
         _LOGGER.debug("Websocket update!")
-        self.parse_sensors()
-        await self.async_parse_sensors()
+        try:
+            await self._update_data_snapshot()
+        except (ValueError, KeyError, UnsupportedFeature) as error:
+            _LOGGER.debug("Error parsing sensors [%s]: %s", type(error).__name__, error)
+            return
+        # Prevent callback failure from stopping future sensor updates; log and continue
+        except Exception as error:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)